.github/workflows/ui-install-and-lint.yml

```diff
@@ -34,14 +34,17 @@ jobs:
           check-latest: true
           always-auth: true
 
+      - name: Enable corepack
```

Comment:
Is there not anything else we need to do to make yarn v4?  This seems...shockingly simple

Reply:
Most of the "work" is in https://github.com/folio-org/stripes-core/pull/1458, but yes, it is shockingly simple. Though, to be fair, a substantial portion of the work amounted to "Wait several years for v2 and v3 to work out the kinks of full backwards compatibility with npm's node_modules dir" and that took a while 😆 .

Reply:
So now we bundle the yarn version inside the repo? Interesting... (although not unlike maven's `mvnw`)